app/Http/Controllers/ProjectCommentController.php

```diff
@@ -50,7 +50,13 @@ public function edit(ProjectComment $comment)
             return redirect('/')->withErrors('You are not authorized to do this action!');
         }
 
-        return view('project_comments.edit', compact('comment'));
+        $route = 'comments';
+        $object = 'message';
+        $routeToDelete = '/comments/'.$comment->id;
```

Comment:
Of course the best solution would be to have the same code everywhere but id this case I don't know which one is the best. Maybe @MichaelCaraccio  could have a more constructed answer.